src/components/calculator/Calculator.test.js

```diff
@@ -15,4 +21,11 @@ describe('Calculator page testing', () => {
     const calcUi = screen.getByText('AC');
     expect(calcUi).toBeInTheDocument();
   });
+
+  test('fireEvent', () => {
+    const { getByText } = render(<Calculator />);
+    const ac = getByText('AC');
+    fireEvent.click(ac);
+    expect(getByText('AC')).toBeTruthy();
+  });
```

Comment:
- [ ] @JakariaJishan great work adding a test for your interaction, it would be nice to add more tests to the interaction. (this is in line with the previous request to add tests for interaction).